web/html/src/components/FormulaForm.js

```diff
@@ -2,6 +2,7 @@
 const React = require("react");
 const Network = require("../utils/network");
 const Functions = require("../utils/functions");
+const jexl = require("jexl");
```

Comment:
To distinguish better I'd Capitalize it to `Jexl` here and everywhere it is used